test/unified_malloc_framework/memoryPool.hpp

```diff
@@ -57,30 +61,243 @@ TEST_P(umfPoolTest, allocFree) {
     umfPoolFree(pool.get(), ptr);
 }
 
-TEST_P(umfPoolTest, pow2AlignedAlloc) {
-#ifdef _WIN32
-    // TODO: implement support for windows
-    GTEST_SKIP();
-#endif
+TEST_P(umfPoolTest, reallocFree) {
+    static constexpr size_t allocSize = 64;
+    static constexpr size_t multiplier = 3;
+    auto *ptr = umfPoolMalloc(pool.get(), allocSize);
+    ASSERT_NE(ptr, nullptr);
+    auto *new_ptr = umfPoolRealloc(pool.get(), ptr, allocSize * multiplier);
+    umf_result_t ret = umfPoolGetLastAllocationError(pool.get());
+    if (ret == UMF_RESULT_ERROR_NOT_SUPPORTED) {
+        GTEST_SKIP();
```

Comment:
Shouldn;t you use the `is*supporeted` function here ? E.g. `if (!isReallocSupported(pool.get()) GTEST_SKIP())` ? I think it would make the rest of the test a bit cleaner.

Reply:
Done. I've used it for all the realloc/calloc tests.